src/Clock.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-console */
+import { Component } from 'react';
+
+export interface Props {
+  name: string;
+}
+
+export interface State {
```

Comment:
```suggestion
interface Props {
  name: string;
}

interface State {
```
Why do you need to export this interfaces?